### ISSUE TITLE
Violating VR definition

### ISSUE THREAD

**Issue by spandan-roy-ainostics** [QIICR/dcmqi#453]
I am trying to create a dicom seg file using itkimage2segimage, however I keep running into VR definition violation errors.
![image](https://user-images.githubusercontent.com/87641486/159239461-b4933293-9b59-48be-aa96-6df3e73ae3f6.png)
![image (1)](https://user-images.githubusercontent.com/87641486/159239667-790cf436-3509-44f1-8238-aafc3532f18d.png)

What would be the exact fix for this issue?

**Comment by fedorov** [comment]
Can you try checking if the input DICOM files are valid using this tool: https://www.dclunie.com/dicom3tools/dciodvfy.html ?

**Comment by spandan-roy-ainostics** [comment]
I am getting the following output when I check the validity for the dicom files:

<img width="1370" alt="image" src="https://user-images.githubusercontent.com/87641486/159888740-3c8bc205-6aab-4da1-8732-d26794872461.png">

Does this mean the images don't meet the validity required by itkimage2segimage?

**Comment by fedorov** [comment]
Yes, you would need to fix the errors related to VR constraints violations before you can use this dataset with dcmqi. Some of those fields are copied into the output SEG, and the converter will not be able to write a valid SEG with those values in the input.
